backend/src/index.ts

```diff
@@ -19,7 +19,8 @@ const httpServer = createServer(app);
 httpServer.listen(3000);
 
 SocketServer.initialize(httpServer);
-const newPokerTable = PokerTable.createPokerTable('table_1', 2);
+
+const newPokerTable = PokerTable.createPokerTable('table_1', 2); // TODO: remove
```

Comment:
I think we need to create the PokerTable from the GameLobbyService class instead of from the pokerTable class.

```suggestion
import GameLobbyService from './game-lobby-service';

GameLobbyService.createPokerTable('table_1', 2); // TODO: remove
```

Reply:
Ah, good catch! I had removed this but didn't commit.  

The real issue is that it this was creating the table directly from the poker table -> creating the room and then erroring on the creation of if inside of the GameLobbyService (if you run `npm run start:debug` you will see the error.

We can actually just want remove it all together as we are creating the table in the GameLobbyService [here] already (https://github.com/richardaspinall/poker-react-node/pull/35/files#diff-a50dc74f7bfca91d5e66c8d5240132342d5351545053c840a637d6f82d378988L13) 

```
[0]   VITE v4.4.11  ready in 127 ms
[0] 
[0]   ➜  Local:   http://127.0.0.1:5173/
[0]   ➜  Network: use --host to expose
[1]   APP:info Socket server initialized +0ms
[1]
[1]   APP:error Room already exists +0ms <<<<<<<--
[1]
[1]   APP:info Xy7hUDJ7F13fAVggAAAC connected +109ms
[1]   APP:info OuDnTruHcNfbczhrAAAD connected +1ms
```